CMakeLists.txt

```diff
@@ -178,14 +179,14 @@ endif()
 
 # Make sure the dylib install name path is set on OSX so you can include dumb in app bundles
 IF(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-    set_target_properties(dumb PROPERTIES INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib)
+    set_target_properties(dumb PROPERTIES INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
```

Comment:
Not sure what the defaults values are for OSX, but here on Linux I think that would evaluate to /usr/usr/lib64 on my system, since `CMAKE_INSTALL_PREFIX` is /usr and `CMAKE_INSTALL_LIBDIR` is /usr/lib64.

Reading [this](https://public.kitware.com/pipermail/cmake/2012-February/049206.html) it sounds like using `CMAKE_INSTALL_LIBDIR` alone would be better, since it's automatically prepended with `CMAKE_INSTALL_PREFIX` if it's a relative path (which it apparently is by default with GNUInstallDirs), and that still lets people pass an absolute custom `CMAKE_INSTALL_LIBDIR`.

Reply:
Indeed. I did not consider CMAKE_INSTALL_LIBDIR to be set to an absolute path in this case (which seems to be done on Fedora, which was my initial motivation for the path-changing a few lines above)
I will use CMAKE_INSTALL_FULL_LIBDIR, which is what we want in this case.

Reply:
fixed in new commit